modules/ctl/src/main/scala/free/ctl.scala

```diff
@@ -53,7 +53,8 @@ object ctl {
         case None    =>
           o.lines.traverse(error(_))      *>
           error(s"exited (${o.exitCode})") *>
-          exit[A](o.exitCode)
+          error(s"Requirement failed. Exiting.") *>
+          exit[A](-1)
```

Comment:
This fails the deployment if the output of a shell command doesn't match what was expected. Like, if we expected one line of output but we got two. Before it only failed if the command itself failed, which wasn't good enough. The point here is that ultimately `runMain` will exits abnormally in this case, signaling a failed build.